subsys/bluetooth/audio/tbs_internal.h

```diff
@@ -357,7 +357,6 @@ struct bt_tbs_instance {
 	struct bt_gatt_subscribe_params termination_sub_params;
 	struct bt_gatt_discover_params termination_sub_disc_params;
 	struct bt_gatt_read_params read_params;
-	uint8_t read_buf[BT_ATT_MAX_ATTRIBUTE_LEN];
```

Comment:
I'm worried that it is a bug that it isn't used. The `net_buf` right below will need a buffer to store the data when performing long reads, and I don't see that being done anywhere. 

We don't have CI tests for TBS to verify this, so I think it may have been broken for a while. 

Can you try to do `bt_tbs_client_read_current_calls` and see if it works?

Reply:
That makes sense. I will check